database.py

```diff
@@ -0,0 +1,53 @@
+"""
+Contains definitions of database models.
+"""
+
+
+from flask_sqlalchemy import SQLAlchemy
+
+db = SQLAlchemy()
+
+
+class BTUser(db.Model):
+    """Database model for BotTrition users."""
+
+    __tablename__ = "btuser"
+
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(80), unique=True, nullable=False)
+    password_hash = db.Column(db.String(80), nullable=False)
+
```

Comment:
We could add more specificity to the restriction if needed, or we could do this later. Maybe add religious_restriction or diet restrictions like keto, vegan, or pescatarian. However, though this sprint covers the MVP, these extra fields are unnecessary and are up to a design decision and can be implemented during the polish sprint.